Source/WebCore/rendering/RenderTheme.cpp

```diff
@@ -873,32 +851,6 @@ bool RenderTheme::paint(const RenderBox& box, ControlStates& controlStates, cons
         updateControlStatesForRenderer(box, controlStates);
         Theme::singleton().paint(appearance, controlStates, paintInfo.context(), devicePixelSnappedRect, box.style().effectiveZoom(), &box.view().frameView(), deviceScaleFactor, pageScaleFactor, box.document().useSystemAppearance(), box.useDarkAppearance(), box.style().effectiveAccentColor());
```

Comment:
I'm not sure if it's right to call this code unconditionally? This was not being called before on iOS for example.

Reply:
Thanks for the quick look @pxlcoder ! I was just throwing it at the bots to see what happened hence the draft part.

Readjusted the patch to basically have `USE(NEW_THEME)` -> `!PLATFORM(IOS_FAMILY)`. Will see if the bots prefer that.

Reply:
Thanks!

cc @annevk who was also interested in doing some cleanup here. Separately from this patch, we should figure out how we can also get rid the divergence between our own platforms.